misty.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.rdoc_options = ['--charset=UTF-8']
   spec.extra_rdoc_files = %w[README.md LICENSE.md]
 
-  spec.add_dependency 'json', '~> 2.0'
+  spec.add_dependency 'json', '>= 1.8.3', '< 3'
```

Comment:
`< 3` is in fact not a change. The previous `~> 2.0` is a shorthand for `>= 2.0, < 3`. I just decreased the minimum requirement; the upper bound stays the same.